xo.config.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+
+const globals = require("globals");
+const { defineConfig } = require("eslint/config");
+const compatPlugin = require("eslint-plugin-compat");
+
+module.exports = defineConfig([
+  {
+    extends: [compatPlugin.configs["flat/recommended"]],
+    languageOptions: {
+      sourceType: "commonjs",
+      globals: {
+        ...globals.browser,
+        ...globals.jquery,
+      },
+    },
+    prettier: true,
+    space: 2,
+    ignores: ["**/vendor/**"],
```

Comment:
This is redundant now, but I added for consistency. We can drop since it's included in the default ignores.